api-documentation.js

```diff
@@ -338,8 +401,55 @@ class ApiDocumentation extends AmfHelperMixin(LitElement) {
     if (old === value) {
       return;
     }
-    this._selectedType = value;
     this.__amfChanged();
+
+    if (!this.handleNavigationEvents) {
+      this._updateServers();
+    }
+
+    this._selectedType = value;
+  }
+
+  get selectedServerValue() {
+    return this._selectedServerValue;
+  }
+
+  set selectedServerValue(value) {
+    const old = this._selectedServerValue;
+    /* istanbul ignore if */
+    if (old === value) {
+      return;
+    }
+    this._selectedServerValue = value;
+    this.requestUpdate('selectedServerValue', old);
+  }
+
+  get selectedServerType() {
+    return this._selectedServerType;
+  }
+
+  set selectedServerType(value) {
```

Comment:
Similarly, this setter/getter does nothing. Use `properties` definition instead.